### PR TITLE
An 524/account for internal txns nick

### PR DESCRIPTION
**Pull request by nickpayiatis**
-Added column inner_tx to specify whether transactions were inner transactions or not
-broke out inner transactions into their own transactions to account for the changes in the 2.8 indexer.
-resolved appl transactions where asset_id was being set as app_id when this type of transaction does not have a asset.

### CODE REVIEW

**Review by juls858 — CHANGES_REQUESTED** [review]
this is not built on 2_8_0.. please refactor and use the new bronze schema.

File: models/algorand/silver/silver_algorand__transactions.sql [hunk 1]
```diff
@@ -32,16 +105,13 @@ SELECT
...
-  ) }}
-  b
+  all_txn b
   LEFT JOIN {{ ref('silver_algorand__transaction_types') }}
   csv
-  ON b.typeenum = csv.typeenum
+  ON b.tx_type = csv.type
 WHERE
-  1 = 1
+  genisis_hash IS NOT NULL
```
[the remark this answers is not in the document]
Comment (nickpayiatis) [reply models/algorand/silver/silver_algorand__transactions.sql]: Conversation for future reference: https://flipsidecrypto.slack.com/archives/C015ULZ1Y7L/p1641413381331900

[the remark this answers is not in the document]
Comment (nickpayiatis) [reply models/algorand/silver/silver_algorand__transactions.sql]: All transaction tables have been modified so we are no longer breaking out inner transactions from their parent transactions. Instead we are using the txn's with no txid(these are actually the inner transactions) and we are tying in the TXID from their parent transactions with the inner_txids model. The inner_txids model should be scheduled before the other silver models.

**Review by juls858 — CHANGES_REQUESTED** [review]
incremental loads are failing...
```
Completed with 7 errors and 0 warnings:

Database Error in model silver_algorand__application_call_transaction (models/algorand/silver/silver_algorand__application_call_transaction.sql)
  002028 (42601): SQL compilation error:
  ambiguous column name '_FIVETRAN_SYNCED'

Database Error in model silver_algorand__asset_transfer_transaction (models/algorand/silver/silver_algorand__asset_transfer_transaction.sql)
  002028 (42601): SQL compilation error:
  ambiguous column name '_FIVETRAN_SYNCED'

Database Error in model silver_algorand__transactions (models/algorand/silver/silver_algorand__transactions.sql)
  002028 (42601): SQL compilation error:
  ambiguous column name '_FIVETRAN_SYNCED'

Database Error in model silver_algorand__asset_freeze_transaction (models/algorand/silver/silver_algorand__asset_freeze_transaction.sql)
  002028 (42601): SQL compilation error:
  ambiguous column name '_FIVETRAN_SYNCED'

Database Error in model silver_algorand__payment_transaction (models/algorand/silver/silver_algorand__payment_transaction.sql)
  002028 (42601): SQL compilation error:
  ambiguous column name '_FIVETRAN_SYNCED'

Database Error in model silver_algorand__key_registration_transaction (models/algorand/silver/silver_algorand__key_registration_transaction.sql)
  002028 (42601): SQL compilation error:
  ambiguous column name '_FIVETRAN_SYNCED'

Database Error in model silver_algorand__asset_configuration_transaction (models/algorand/silver/silver_algorand__asset_configuration_transaction.sql)
  002028 (42601): SQL compilation error:
  ambiguous column name '_FIVETRAN_SYNCED'

Done. PASS=15 WARN=0 ERROR=7 SKIP=7 TOTAL=29```

**Review by juls858 — APPROVED** [review]

**Pull request merged**